src/families/cosmos/shared/02-SelectAmount.js

```diff
@@ -257,7 +278,6 @@ const styles = StyleSheet.create({
     flex: 1,
     alignItems: "center",
     justifyContent: "center",
-    minHeight: 150,
```

Comment:
Leave this one here without that the whole screen collapses on itself on android :sweat_smile:

Reply:
ah, that makes sense